plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -94,29 +95,31 @@
      */
     public static final String DOCKER_IMAGE_TYPE = "image";
 
-    private final DockerConnector                  docker;
-    private final DockerInstanceStopDetector       dockerInstanceStopDetector;
-    private final DockerContainerNameGenerator     containerNameGenerator;
-    private final WorkspaceFolderPathProvider      workspaceFolderPathProvider;
-    private final boolean                          doForcePullOnBuild;
-    private final boolean                          privilegeMode;
-    private final Set<String>                      supportedRecipeTypes;
-    private final DockerMachineFactory             dockerMachineFactory;
+    private final DockerConnector                               docker;
+    private final UserSpecificDockerRegistryCredentialsProvider dockerCredentials;
+    private final DockerInstanceStopDetector                    dockerInstanceStopDetector;
+    private final DockerContainerNameGenerator                  containerNameGenerator;
+    private final WorkspaceFolderPathProvider                   workspaceFolderPathProvider;
+    private final boolean                                       doForcePullOnBuild;
+    private final boolean                                       privilegeMode;
+    private final Set<String>                                   supportedRecipeTypes;
+    private final DockerMachineFactory                          dockerMachineFactory;
```

Comment:
Only part of fields are aligned. Align all of them or move too long field in a separate block to avoid change in aligning